platform/docs/docs/migration-guide.md

```diff
@@ -0,0 +1,492 @@
+---
+sidebar_position: 10
+sidebar_label: Migration Guide
+---
+
+# Migration Guide
+
+On this page, we will provide a guide to migrating from OHIF v2 to v3. Please note
+that this document is a work in progress and will be updated as we move forward.
+
+
+## Summary of Changes
+
+OHIF v3 is a complete re-architecture of the OHIF v2 to make it more modular and
+easier to maintain. The main differences are:
+
+- Extensions don't inject their modules into the viewer, they will make them available
+  to be used by the viewer.
+- To use the modules provided by the extensions, you need to write a [Modes](./platform/modes/index.md). Modes
+are configuration objects that will be used by the viewer to load the modules.
+- App configuration structure is different, mainly the `servers` is renamed to `dataSources`.
+- The viewer UI is completely re-written in tailwindcss for better maintainability.
+- cornerstone-core and cornerstone-tools are deprecated and OHIF v3 is using the new Cornerstone3D rendering library and tools
+- A new CLI tool to help you create extensions and modes.
```

Comment:
Maybe its a good idea to add links to the official documentation where needed? e.g. extensions / modes or cli